10x/benchmark/10x_cost_analysis.ipynb

```diff
@@ -0,0 +1,295 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import os\n",
+    "import json\n",
+    "from datetime import datetime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook expects a single environment variable, `METADATA`, to be set to the location of the metadata file you want processed. If running interactively, uncomment the below cell to set this parameter within the notebook environment. \n",
+    "\n",
+    "This notebook will create three plots: \n",
+    "\n",
+    "1. an unmodified cost file (not saved by default)\n",
+    "2. cost per task, merging split/main/join tasks with the same prefix\n",
+    "3. run time per task, merging split/main/join tasks with the same prefix\n",
+    "\n",
+    "In addition, it will print the total run time in wall clock hours and cpu hours, as well as the total cost across all tasks. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# to run interactively, uncomment the line below\n",
+    "# os.environ['METADATA] = 'path_to_metadata.json'"
```

Comment:
Is this a typo? I assume it would be `os.environ['METADATA'] = './metadata.json'`

Reply:
Yes! great catch. I'll fix this.